pymc3/step_methods/metropolis.py

```diff
@@ -510,8 +510,8 @@ class DEMetropolis(PopulationArrayStepShared):
         S (and n). Defaults to Uniform(-S,+S).
     scaling : scalar or array
         Initial scale factor for epsilon. Defaults to 0.001
-    tune : bool
-        Flag for tuning the scaling. Defaults to True.
+    tune : str
+        Which hyperparameter to tune. Defaults to None, but can also be 'scaling' or 'lambda'.
```

Comment:
Is it possible to do both?

Reply:
Tuning both at the same time could lead to weird swing-in slingshots that are hard to understand.

I think either tuning option has their corner cases (`scaling` when the posterior is very narrow, `lambda` on multi-modality or banana-shapes), but on those corner-cases one should just do extensive tuning first, and then fix the initial values to the tuning result by hand.